react/react1/week1/week1hw-simple-list/src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```

Comment:
Just push the files that you have worked on, no need to push these all files fx json files, test files, images that you didnot uploaded, gitignore files, and many other